src/context/Panel.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
nit: different file name casing on this and a few other files. intentional?

Reply:
Classes in upper. Everything else in lower. Minor exceptions.

Your team prefers all lower, is that right?

Reply:
yeah that's what we've been doing since almost everything is encapsulated in a class =P

---
In reply to: [434751991](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r434751991) [](ancestors = 434751991)